### ISSUE TITLE
Can not set Height to options

### ISSUE THREAD

**Issue by Xing-He** [hiNISAL/any-progress#1]
![image](https://user-images.githubusercontent.com/11990205/68108404-a1cc4300-ff22-11e9-93f3-a6572ad8ee1c.png)

**Comment by hiNISAL** [comment]
Sorry🙏, this problem has been fixed(version 1.4.3).

If you cannot update whit some problem(network and more) by npm, you can use the `style` options.

``` js
new AnyProgress({
    style: {
        height: '10px',
    },
});
```

Thanks to provide this problem, and i will add the test case to this project.😄

**Comment by Xing-He** [comment]
👌， really helpful
